--- units-1.88/Makefile.in.orig	2008-04-22 13:07:04.000000000 +0200
+++ units-1.88/Makefile.in	2010-02-16 13:35:38.000000000 +0100
@@ -44,7 +44,7 @@
 
 transform=@program_transform_name@
 
-DEFS = -DUNITSFILE=\"@UDAT@units.dat\" @DEFIS@ @DEFS@
+DEFS = -DUNITSFILE=\"$(datadir)/misc/units.dat\" @DEFIS@ @DEFS@
 CFLAGS = @CFLAGS@
 OBJECTS = units.@OBJEXT@ parse.tab.@OBJEXT@ getopt.@OBJEXT@ getopt1.@OBJEXT@ @STRFUNC@
 
@@ -78,14 +78,14 @@
 	$(CC) $(LDFLAGS) -o units $(OBJECTS) $(LIBS)
 
 install: units@EXEEXT@ units.dat install-doc
-	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)/misc
 	$(INSTALL_PROGRAM) units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/misc/units.dat
 
 install-strip: units@EXEEXT@ units.dat install-doc
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
 	$(INSTALL_PROGRAM) -s units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/misc/units.dat
 
 install-doc: install-man install-info
 
@@ -97,7 +97,7 @@
 	./texi2man units.texinfo > units.man
 
 units.1: units.man
-	sed s!@DATAFILE@!@UDAT@units.dat! $(srcdir)/units.man > units.1
+	sed s!@DATAFILE@!$(datadir)/misc/units.dat! $(srcdir)/units.man > units.1
 
 install-info: $(DESTDIR)$(infodir)/units.info
 
@@ -127,7 +127,7 @@
 uninstall: 
 	-rm -f $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'` \
 	       $(DESTDIR)$(mandir)/man1/`echo units|sed '$(transform)'`.1 \
-	       $(DESTDIR)$(infodir)/units.info $(DESTDIR)$(datadir)/units.dat
+	       $(DESTDIR)$(infodir)/units.info $(DESTDIR)$(DESTDIR)$(datadir)/misc/units.dat
 	@if test -f $(DESTDIR)$(infodir)/dir; then \
 	  echo You may need to remove units.info from $(DESTDIR)$(infodir)/dir; \
 	else true; fi
