--- units-2.24/Makefile.in.orig	2025-01-21 18:59:19.460359728 +0100
+++ units-2.24/Makefile.in	2025-01-21 19:48:30.781037718 +0100
@@ -61,7 +61,7 @@ DEFS=-DUNITSFILE=\"@UDATADIR@definitions
                     @DEFIS@ @DEFS@
 CFLAGS = @CFLAGS@
 CPPFLAGS = @CPPFLAGS@
-OBJECTS = units.@OBJEXT@ parse.tab.@OBJEXT@ getopt.@OBJEXT@ getopt1.@OBJEXT@ @STRFUNC@
+OBJECTS = units.@OBJEXT@ parse.tab.@OBJEXT@ @STRFUNC@
 
 .PHONY: currency-update-check dist sig validate-codata
 
