--- units-1.80/Makefile.in~	Tue Apr 29 11:48:16 2003
+++ units-1.80/Makefile.in	Tue Apr 29 11:48:53 2003
@@ -45,7 +45,7 @@
 transform=@program_transform_name@
 
 datadir = @datadir@
-DEFS = -DUNITSFILE=\"@UDAT@units.dat\" @DEFIS@ @DEFS@
+DEFS = -DUNITSFILE=\"$(datadir)/misc/units.dat\" @DEFIS@ @DEFS@
 CFLAGS = @CFLAGS@
 ALL_CFLAGS = $(DEFS) $(CFLAGS) -I$(srcdir)
 COMPILE = $(CC) $(ALL_CFLAGS) $(LDFLAGS)
@@ -95,13 +95,13 @@
 	$(CC) $(LDFLAGS) -o units $(OBJECTS) parse.tab.o @STRFUNC@ $(LIBS)
 
 install: units units.dat install-doc
-	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)/misc
 	$(INSTALL_PROGRAM) units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/misc/units.dat
 
 install-strip: units units.dat install-doc
 	$(INSTALL_PROGRAM) -s units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/misc/units.dat
 
 install-doc: install-man install-info
 
@@ -113,7 +113,7 @@
 	./texi2man units.texinfo > units.man
 
 units.1: units.man
-	sed s!@DATAFILE@!@UDAT@units.dat! $(srcdir)/units.man > units.1
+	sed s!@DATAFILE@!$(datadir)/misc/units.dat! $(srcdir)/units.man > units.1
 
 install-info:
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(infodir)
@@ -125,7 +125,7 @@
 uninstall: 
 	-rm -f $(DESTDIR)$(DESTDIR)$(bindir)/`echo units|sed '$(transform)'` \
 	       $(DESTDIR)$(mandir)/man1/`echo units|sed '$(transform)'`.1 \
-	       $(DESTDIR)$(infodir)/units.info $(DESTDIR)$(DESTDIR)$(datadir)/units.dat
+	       $(DESTDIR)$(infodir)/units.info $(DESTDIR)$(DESTDIR)$(datadir)/misc/units.dat
 	@if test -f $(DESTDIR)$(infodir)/dir; then \
 	  echo You may need to remove units.info from $(DESTDIR)$(infodir)/dir; \
 	else true; fi
