--- units-1.80/Makefile.in.orig	Fri May  3 03:15:08 2002
+++ units-1.80/Makefile.in	Tue Apr 29 11:23:30 2003
@@ -95,19 +95,19 @@
 	$(CC) $(LDFLAGS) -o units $(OBJECTS) parse.tab.o getopt.o getopt1.o @STRFUNC@ $(LIBS)
 
 install: units units.dat install-doc
-	$(srcdir)/mkinstalldirs $(bindir) $(datadir)
-	$(INSTALL_PROGRAM) units $(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(datadir)/units.dat
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
+	$(INSTALL_PROGRAM) units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
 
 install-strip: units units.dat install-doc
-	$(INSTALL_PROGRAM) -s units $(bindir)/`echo units|sed '$(transform)'`
-	$(INSTALL_DATA) $(srcdir)/units.dat $(datadir)/units.dat
+	$(INSTALL_PROGRAM) -s units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
+	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
 
 install-doc: install-man install-info
 
 install-man: units.1
-	$(srcdir)/mkinstalldirs $(mandir)/man1
-	$(INSTALL_DATA) units.1 $(mandir)/man1/`echo units|sed '$(transform)'`.1
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/man1
+	$(INSTALL_DATA) units.1 $(DESTDIR)$(mandir)/man1/`echo units|sed '$(transform)'`.1
 
 units.man: units.texinfo texi2man
 	./texi2man units.texinfo > units.man
@@ -115,36 +115,19 @@
 units.1: units.man
 	sed s!@DATAFILE@!@UDAT@units.dat! $(srcdir)/units.man > units.1
 
-install-info: $(infodir)/units.info
-
-$(infodir)/units.info: units.info
-	$(srcdir)/mkinstalldirs $(infodir)
-# There may be a newer info file in . than in srcdir.
-	-if test -f units.info; then d=.; \
-	   else d=$(srcdir); fi; \
-	$(INSTALL_DATA) $$d/units.info $@; \
-# Run install-info only if it exists.
-# Use `if' instead of just prepending `-' to the
-# line so we notice real errors from install-info.
-# We use `$(SHELL) -c' because some shells do not
-# fail gracefully when there is an unknown command.
-	if test -f $(infodir)/dir; then \
-	  if $(SHELL) -c 'install-info --version' \
-	     >/dev/null 2>&1; then \
-	     install-info --dir-file=$(infodir)/dir \
-	               $(infodir)/units.info; \
-	  else true; fi \
-	else true; fi
+install-info:
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(infodir)
+	$(INSTALL_DATA) units.info $(DESTDIR)$(infodir)
 
 Makefile: Makefile.in config.status
 	./config.status
 
 uninstall: 
-	-rm -f $(bindir)/`echo units|sed '$(transform)'` \
-	       $(mandir)/man1/`echo units|sed '$(transform)'`.1 \
-	       $(infodir)/units.info $(datadir)units.dat
-	@if test -f $(infodir)/dir; then \
-	  echo You may need to remove units.info from $(infodir)/dir; \
+	-rm -f $(DESTDIR)$(DESTDIR)$(bindir)/`echo units|sed '$(transform)'` \
+	       $(DESTDIR)$(mandir)/man1/`echo units|sed '$(transform)'`.1 \
+	       $(DESTDIR)$(infodir)/units.info $(DESTDIR)$(DESTDIR)$(datadir)/units.dat
+	@if test -f $(DESTDIR)$(infodir)/dir; then \
+	  echo You may need to remove units.info from $(DESTDIR)$(infodir)/dir; \
 	else true; fi
 
 clean mostlyclean: 
